src/ssh/test-sshbridge.c

```diff
@@ -299,9 +300,11 @@ setup (TestCase *tc,
 
       content = g_strdup_printf ("Host some_host\n "
                                  "\tHostname %s\n"
-                                 "\tPort %s\n",
+                                 "\tPort %s\n"
+                                 "\tUser %s\n",
                                  host_port[0],
-                                 g_strv_length (host_port) > 1 ? host_port[1] : "22");
+                                 g_strv_length (host_port) > 1 ? host_port[1] : "22",
+                                 fixture->ssh_config_user ? fixture->ssh_config_user : g_get_user_name ());
```

Comment:
I'd rather the template was just `%s` and if the fixture sets a user name, append the entire `\tUser %s` bit. I. e. if the fixture does not specify a user name, don't set one at all in the config. We want to test that default case as well. And then have a new test with a valid user name (i. e. g_get_user_name ())

Reply:
ack